build/checkstyle/tachyon_checks.xml

```diff
@@ -47,6 +47,11 @@
     <property name="fileExtensions" value=".java"/>
   </module>
   <module name="RegexpSingleline">
+    <property name="format" value="TODO[^\(]"/>
```

Comment:
I see. Unfortunately checkstyle doesn't support regex matching only in comments. I think having TODO in code is very likely, and we can always revisit this rule when we find a false positive.